test/it/uk/gov/hmrc/exports/base/IntegrationTestPurgeSubmissionsToolSpec.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmrc.exports.base
+
+import org.mongodb.scala.MongoCollection
+import org.mongodb.scala.bson.BsonDocument
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import uk.gov.hmrc.exports.repositories.{
+  DeclarationRepository,
+  ParsedNotificationRepository,
+  SubmissionRepository,
+  UnparsedNotificationWorkItemRepository
+}
+
+trait IntegrationTestPurgeSubmissionsToolSpec extends IntegrationTestSpec with GuiceOneAppPerSuite {
+
+  val submissionRepository = app.injector.instanceOf[SubmissionRepository]
+  val declarationRepository = app.injector.instanceOf[DeclarationRepository]
+  val notificationRepository = app.injector.instanceOf[ParsedNotificationRepository]
+  val unparsedNotificationRepository = app.injector.instanceOf[UnparsedNotificationWorkItemRepository]
```

Comment:
If you use `IntegrationTestBaseSpec` these can be semplified to `instanceOf[...]`